index.html

```diff
@@ -423,6 +423,28 @@ <h2>
                       abort these steps. 
```

Comment:
The "resolve and abort these steps" bullet right before the new bullets, seems like it ensure they'd never be reached.